src/components/Comment/Comment.tsx

```diff
@@ -144,6 +145,8 @@ const svgOverrides = css`
         fill: ${neutral[46]} !important;
         left: 3px !important;
         bottom: 0 !important;
+        width: 18px !important;
+        height: 18px !important;
```

Comment:
If we still need to override the position and size of the icon then something is wrong. The point of using a Source icon was to remove the need for the overrides